src/noir/common/log.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include <fc/log/appender.hpp>
+#include <fc/log/log_message.hpp>
+#include <fc/log/logger_config.hpp>
+
+namespace noir::log {
+
+  class tmlog_appender : public fc::appender {
+  public:
+    explicit tmlog_appender( const fc::variant& args );
+    tmlog_appender();
+
+    virtual ~tmlog_appender();
+
+    virtual void initialize( boost::asio::io_service& io_service ) override;
```

Comment:
- Using virtual and override keywords at the same time is not meaningful. [[the C++ Core Guidelines C.128]](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final)
- `boost::asio::io_service` is deprecated and renamed to `boost::asio::io_context`.

```suggestion
    void initialize(boost::asio::io_context& io_context) override;
```

Reply:
Fixed